identity/manager.go

```diff
@@ -45,23 +45,23 @@ func (idm *identityManager) GetIdentities() []dto.Identity {
 
 	var ids = make([]dto.Identity, len(accountList))
 	for i, account := range accountList {
-		ids[i] = *accountToIdentity(account)
+		ids[i] = accountToIdentity(account)
 	}
 
 	return ids
 }
 
-func (idm *identityManager) GetIdentity(identityString string) *dto.Identity {
+func (idm *identityManager) GetIdentity(identityString string) dto.Identity {
```

Comment:
`GetIdentity() (dto.Identity, error)` to recognise error about that result is empty